src/use-mutation.ts

```diff
@@ -55,6 +55,8 @@ export interface UseMutationOptions<
    */
   mutation: (vars: TVars, context: NoInfer<TContext>) => Promise<TResult>
 
+  key?: MaybeRefOrGetter<UseEntryKey>
```

Comment:
In mutations, it should be able to access the variables passed to the mutation, similar to the invalidate keys

I'm realizing that allowing a computed property or a ref is actually problematic as it could create the issues highlighted in #11. It's something we will have to look into later on

Reply:
I am wondering, for the ref or computed properties, is it then really worth handling them? I am just realising that users would probably rather trigger a mutation imperatively than declaratively relying on reactivity, and therefore I am wondering if passing reactivity through the `key` (and the issue associated) is enough profitable. Maybe I could just type the `key` the same way as you did for the `keys` property (without reactivity).

Reply:
You are right; passing a ref or a computer seems like a foot-gun. It should only be static or computed from the arguments